usco/Jam
# Fancy scrolling bom

It adds the adder to the header table and ads scrolling behaviour to the rest of the bom. Also I added column-names to the cells classes so we can more easily edit the widths the columns of both tables. 

## How to test: 
- open the the jam test app
- add so many files a scrollbar appears
- resize the window and see what it does
- also test other bom behaviour like what happens when you select a part, delete it, etc. 
- perform all previous tests also in readonly mode
